src/elm/Page/Register.elm

```diff
@@ -557,6 +554,11 @@ pdfData keys =
     }
 
 
+dummyCountry : Address.Country
+dummyCountry =
```

Comment:
I don't think this is a good idea, we can know from the invitation what is the community, and thus know if it have KYC enabled, we can do better!